### PR TITLE
Cleanup package.json

### PR DESCRIPTION
**Pull request by jden**
- Adds [SPDX](http://npm1k.org/) license code for apache-2.0 license
- Sets Zendesk, Inc. as the author (move mixu to contributors list)
- Adds some keywords because SEO like it's 1998

Required
========
- [ ] :+1: from @zendesk/radar

Risks
========
- none

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,11 +20,16 @@
     {
       "name": "Nicolas Herment",
       "url": "https://github.com/nherment"
-    }
+    },
+    "jden <jason@denizac.org>"
   ],
-  "main": "index.js",
```
Comment (kazraza) [inline package.json]: Why is this removed, is primary entry point different?

**Comment by kazraza** [comment]
:+1:

File: package.json (hunk 1 above)
Comment (kazraza) [inline package.json]: Thanks @jden for answering offline.

Comment (jden) [inline package.json]: no problem! for documentation's sake:

"main" in package.json sets the default package entry point, e.g. when other packages `require('radar')`. The default is `index.js`, so this change removes the redundant explicit declaration.
